Microsoft.Research/Contracts/WindowsBase/System.Windows.Rect.cs

```diff
@@ -16,6 +16,19 @@
 using System.ComponentModel;
 using System.Diagnostics.Contracts;
 
+
+// 2015-03-36: tom-englert
```

Comment:
:bulb: I prefer these types of longer comment stay in the issue instead of the code. I've found  that over time, code gets corrected but the comments tend to stay forever.

:memo: The other inline comments you added at the location of commented calls to `Contract.Requires` below are different. They are short, and obviously would be changed if someone did implement support for these contracts in the future.